use-cases/single-file-CC-program/test-main.cc

```diff
@@ -32,27 +34,33 @@ timespec_to_ns(struct timespec *ts)
 }
 
 int
-main(void)
+main(const int argc, const char * argv[])
 {
-    // Logging perf benchmarking methods below share this log-file.
-    auto e = l3_init("/tmp/l3.cc-test.dat");
-    if (e) {
-        abort();
+    const char *logfile = NULL;
+    if (argc == 1) {
```

Comment:
Should we check the usage more carefully? If I'm reading this correctly, we can pass any command-line arguments at all and it will skip perf benchmarking. Which is OK I guess, but a little weird/confusing.

Reply:
Yes, perf benchmarking will only be run with no arguments.

I just wanted to suppress that when this program is invoked with `--unit-tests` argument. So changed the code in the 3 use-case sample programs to only do the `else()` part [i.e., non-perf unit-tests] if `--unit-tests` arg is provided.